kobuki/b166c93.bugzoo.yml

```diff
@@ -0,0 +1,31 @@
+version: '1.0'
+
+blueprints:
+  - type: docker
+    tag: robustrosin/robust:b166c93
+    file: ../Dockerfile
```

Comment:
@ChrisTimperley: I'd like to place the `.bugzoo.yml` file in a subdir of the `kobuki` dir, but if I do that I get errors when trying to build the image:

```shell
user@host:~$ bugzoo bug build robust:b166c93
building bug: robust:b166c93
Building image: robustrosin/robust:b166c93
Traceback (most recent call last):
  File "/home/user/bugzoo_venv/lib/python3.5/site-packages/bugzoo/mgr/build.py", line 104, in build
    shutil.copy(instructions.file_abs, tf)
  File "/home/user/bugzoo_venv/lib/python3.5/shutil.py", line 235, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/home/user/bugzoo_venv/lib/python3.5/shutil.py", line 115, in copyfile
    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/robust_rosin/kobuki/b166c93/.Dockerfile'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/user/bugzoo_venv/bin/bugzoo", line 11, in <module>
    sys.exit(main())
  File "/home/user/bugzoo_venv/lib/python3.5/site-packages/bugzoo/cli/__init__.py", line 541, in main
    args.func(args)
  File "/home/user/bugzoo_venv/lib/python3.5/site-packages/bugzoo/cli/__init__.py", line 489, in <lambda>
    cmd.set_defaults(func=lambda args: build_bug(rbox, args.bug, args.force))
  File "/home/user/bugzoo_venv/lib/python3.5/site-packages/bugzoo/cli/__init__.py", line 103, in build_bug
    rbox.bugs.build(bug, force=force)
  File "/home/user/bugzoo_venv/lib/python3.5/site-packages/bugzoo/mgr/bug.py", line 83, in build
    quiet=quiet)
  File "/home/user/bugzoo_venv/lib/python3.5/site-packages/bugzoo/mgr/build.py", line 130, in build
    os.remove(tf)
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/robust_rosin/kobuki/b166c93/.Dockerfile'
```

I assumed that was because I only had `../Dockerfile` for `file`, but even changing that to `../../Dockefile` didn't seem to help.